js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js

```diff
@@ -1200,20 +1207,32 @@
         )
       );
 
+      if (window.App.replaceApp) {
+        tableContent.push(
+          window.modalView.createCheckboxEntry(
+            'new-app-flag-teardown',
+            'Run teardown?',
+            true,
+            "Should the existing service's teardown script be executed before replacing the service?",
+            false
+          )
+        );
+      }
+
       tableContent.push(
         window.modalView.createCheckboxEntry(
-          'new-app-teardown',
+          'new-app-flag-setup',
           'Run setup?',
           true,
-          "Should this app's setup script be executed after installing the app?",
+          "Should this service's setup script be executed after installing the service?",
           true
         )
       );
 
       if (window.App.replaceApp) {
         tableContent.push(
           window.modalView.createCheckboxEntry(
```

Comment:
This checkbox always appears pre-checked for me, which is an unexpected change. Can you check why that is and revert it to being unchecked by default? Maybe the two boolean parameters of createCheckboxEntry have been mixed up?

The default behavior should be NOT to replace (i.e. discard config & settings) by default.

Reply:
Done, see latest commit: https://github.com/arangodb/arangodb/pull/9719/commits/bbe55edea591343e9cbcfdc90b6a37a60c763e89